docs/7_0/components/texteditor.md

```diff
@@ -44,6 +44,8 @@ allowImages | true | Boolean | Whether to allow images to be included.
 formats | null | List | Define a list of formats to allow in the editor. By default all formats are allowed.
 
 ## Getting started with the Editor
+!>TextEditor requires the **owasp-java-html-sanitizer** dependency added to your pom.xml to prevent XSS. 
+
```

Comment:
This signifies important content so it is highlighted. See https://docsify.js.org/#/helpers?id=important-content